### DISCUSSION TITLE
Unable to build on gentoo/quickjs

### DISCUSSION THREAD

**Discussion by damobrisbane** [apache/couchdb#5267] [General] [answered]
hi, 101 question, cant "build" on gentoo. Have installed quickjs, can see a library in there. Any help to actually compile? ie:

```
> 
> # apache-couchdb-3.4.1
> 
> find /usr/lib/quickjs/
/usr/lib/quickjs/
/usr/lib/quickjs/libquickjs.a
> 
> ./configure --dev --spidermonkey-version quickjs
ERROR: SpiderMonkey quickjs is not found. Please specify with --spidermonkey-version.
> 
```

thank you!

**Comment by nickva** [comment, marked as answer]
QuickJS is vendored in the CouchDB source code directly so that should compile if gcc + libc is present. However, by default we also try to compile against a version of spidermonkey since that's still the default. There is a mode however to build only with QuickJS.

To try it out can do:

```
./configure --dev --disable-spidermonkey
 make release
```

**Reply by damobrisbane** [reply]
that got pas my error and now building, thanks Nick
